x/protocolpool/README.md

```diff
@@ -6,11 +6,9 @@ sidebar_position: 1
 
 ## Concepts
 
-TODO: update docs to reflect that this module is supplemental
+`x/protocolpool` is a module that handle functionality around community pool funds. This provides a separate module account for community pool making it easier to track the pool assets. We no longer track community pool assets in `x/distribution` module, but instead in the `x/protocolpool` module. Funds are migrated from the `x/distribution` module's community pool to `x/protocolpool`'s module account.
```

Comment:
```suggestion
`x/protocolpool` is a supplemental Cosmos SDK module that handles functionality for community pool funds. The module provides a separate module account for the community pool making it easier to track the pool assets. We no longer track community pool assets in `x/distribution` module, but instead in the `x/protocolpool` module. Funds are migrated from the `x/distribution` module's community pool to `x/protocolpool`'s module account.
```

Reply:
for this part:  "we no longer track community pool assets...", i think we should mention something like, "starting with v0.53.x of the Cosmos SDK, community funds can be tracked using this module" or something of that nature. the way it currently reads, there is some implied contextual knowledge the reader might not have.